sirepo/util.py

```diff
@@ -325,22 +351,6 @@ def json_dump(obj, path=None, pretty=False, **kwargs):
     return res
 
 
-def raise_bad_request(*args, **kwargs):
-    _raise("BadRequest", *args, **kwargs)
-
```

Comment:
There is a call to raise_bad_request in http_request.py which needs to be replaced

Reply:
The code was not executed because content_type_eq was blowing up so the result was a server error. There's a test and this should be fixed.